examples/hif2a/fit_to_multiple_rbfes.py

```diff
@@ -311,8 +339,13 @@ def unflatten(theta: array) -> Dict[Handler, array]:
     np.save(output_path.joinpath('step_indices.npy'), np.hstack(step_inds)[:args.param_updates])
 
     step = 0
+    epoch = 0
     # in each optimizer step, look at one transformation from relative_transformations
-    for epoch, steps in enumerate(step_inds):
+    for steps in step_inds:
+        # Run Validation edges at start of epoch. Unlike NNs we have a reasonable starting
+        # point that is worth knowing
+        run_validation_edges(validation, ordered_params, systems, epoch)
```

Comment:
Running the validation after training made it hard to determine the training without rerunning RBFE calculations.

Reply:
+1, running validation before hand is the standard way to do this.